File: app/src/main/java/ru/nikitazhelonkin/coinbalance/data/api/service/coin/BTCApiService.java
package ru.nikitazhelonkin.coinbalance.data.api.service.coin;


import io.reactivex.Single;
import okhttp3.ResponseBody;
import retrofit2.http.GET;
import retrofit2.http.Path;

public interface BTCApiService {

    @GET("q/addressbalance/{address}")
    Single<ResponseBody> balance(@Path("address") String address);


}

File: app/src/main/java/ru/nikitazhelonkin/coinbalance/data/api/coin/BTCApiService.kt
package ru.nikitazhelonkin.coinbalance.data.api.coin


import io.reactivex.Single
import okhttp3.ResponseBody
import retrofit2.http.GET
import retrofit2.http.Path

interface BTCApiService {

    @GET("q/addressbalance/{address}")
    fun balance(@Path("address") address: String): Single<ResponseBody>

}
